### ISSUE TITLE
Steamwebhelper Not Responding on Whisky 2.3.3 When Launching Steam

### ISSUE THREAD

**Issue by playerpatience** [Whisky-App/Whisky#1203]
### Description

I'm encountering an issue with Whisky (version 2.3.3) while attempting to run Steam.

"steamwebhelper, a critical Steam component, is not responding. The Steam UI will not be usable. Click here for more information."
I have attempted a workaround involving downgrading Steam and using specific launch arguments. While I successfully downgraded Steam, attempting to launch it with steam.exe and the specified arguments does not work; Steam does not open at all.

Interestingly, Steam launches after a fresh reinstall, but this results in the same error message reappearing shortly after.

### Steps to reproduce

1. Open Whisky (version 2.3.3)
2. Create or use an existing bottle with Steam installed (my bottle name is "CS2").
3. Downgrade Steam to an older version.
4. Attempt to launch Steam using the following arguments: **-noverifyfiles -nobootstrapupdate -skipinitialbootstrap -norepairfiles -overridepackageurl**

### Expected behaviour

Steam should open successfully and function without error.

### Logs

Whisky Version: 2.3.3
Date: 2024-11-07T15:20:01Z
macOS Version: 14.6.1

Bottle Name: CS2
Bottle URL: /Users/[redacted]/Library/Containers/com.isaacmarovitz.Whisky/Bottles/88E71A30-6E66-42FC-BCDD-6E85CE8AECEE

Wine Version: 7.7.0
Windows Version: win11
Enhanced Sync: msync

Metal HUD: false
Metal Trace: false

Arguments: start /unix /Users/[redacted]/Library/Containers/com.isaacmarovitz.Whisky/Bottles/88E71A30-6E66-42FC-BCDD-6E85CE8AECEE/drive_c/Program Files (x86)/Steam/steam.exe -noverifyfiles -nobootstrapupdate -skipinitialbootstrap -norepairfiles -overridepackageurl

Environment:
{
    "DXVK_ASYNC" = 1;
    "GST_DEBUG" = 1;
    WINEDEBUG = "fixme-all";
    WINEESYNC = 1;
    WINEMSYNC = 1;
    WINEPREFIX = "/Users/[redacted]/Library/Containers/com.isaacmarovitz.Whisky/Bottles/88E71A30-6E66-42FC-BCDD-6E85CE8AECEE";
}

msync: bootstrapped mach port on wine-1c1307d-msync.
msync: up and running.
0140:err:ntlm:ntlm_LsaApInitializePackage no NTLM support, expect problems

### What version of Whisky are you using?

2.3.2

### What version of macOS are you using?

Sonoma (macOS 14)

### Issue Language

- [x] Yes my issue is written in English

**Comment by Toast412** [comment]
The new steam update cut something out, https://store.steampowered.com/news/collection/steam/?emclan=103582791457287600&emgid=4472730495692571024. I feel it is best to downgrade steam to the previous release to make it work cause something broke the update

**Comment by playerpatience** [comment]
> The new steam update cut something out, https://store.steampowered.com/news/collection/steam/?emclan=103582791457287600&emgid=4472730495692571024. I feel it is best to downgrade steam to the previous release to make it work cause something broke the update

I did downgrade, however opening steam.exe doesn't seem to work. I've noticed steam opens only after using the setup, which uses the "steam://open" command.

**Comment by PonomareVlad** [comment]
Duplicate of #1202

**Comment by Mxbikesuser** [comment]
How do i downgrade back to a previous version of steam if i can’t get it running??

**Comment by Mxbikesuser** [comment]
> The new steam update cut something out, https://store.steampowered.com/news/collection/steam/?emclan=103582791457287600&emgid=4472730495692571024. I feel it is best to downgrade steam to the previous release to make it work cause something broke the update

How do i even downgrade and go back an update for steam if i can’t get it running? help please

**Comment by ryanholden8** [comment]
> > The new steam update cut something out, https://store.steampowered.com/news/collection/steam/?emclan=103582791457287600&emgid=4472730495692571024. I feel it is best to downgrade steam to the previous release to make it work cause something broke the update
> 
> How do i even downgrade and go back an update for steam if i can’t get it running? help please

Right click on steam app in whiskey (assuming it's pinned) and add the following to the arguments section. Go back and then run steam.

https://github.com/Whisky-App/Whisky/issues/1199#issuecomment-2460454741

**Comment by andymitchell** [comment]
The update broke Steam for me too (M1 MacBook Pro). 

I confirm this works as a temporary workaround by rolling back and blocking the update. (It also fixed the same problem in Porting Kit)... 
 
> Right click on steam app in whiskey (assuming it's pinned) and add the following (see [#1199 (comment)](https://github.com/Whisky-App/Whisky/issues/1199#issuecomment-2460454741)) to the arguments section. Go back and then run steam.

**Comment by itskando** [comment]
Issue occurs when plugging in external monitor. 

Steam fails to open with HDMI plugged in. 
Steam opens fine with HDMI not plugged in 
With Steam open, if HDMI is inserted, steam disappears when display appears on external monitor (mirror mode).

**Comment by itskando** [comment]
Update:

I only have the issue when macOS display mode is mirroring. (I did not need the downgrade fix.)

If I am using the laptop internal monitor or an external monitor without the internal monitor (clamshell mode — not available when using battery power), I do not have this issue. 

If I am using the laptop internal monitor and plug an external monitor HDMI cable in, steam will even close immediately.
